go/vt/sqlparser/ast.go

```diff
@@ -2207,6 +2207,14 @@ type (
 		alternative  Expr // this is what will be used to Format this struct
 	}
 
+	// JSONUtilityExpr represents set of JSON utility functions
+	// More information on https://dev.mysql.com/doc/refman/8.0/en/json-utility-functions.html
+	JSONUtilityExpr struct {
+		Name      ColIdent
+		StringArg Expr
+		Column    *ColName
+	}
```

Comment:
The name should have been an enum. I would rather prefer having different structs for all the three functions, since they have very little in common other than being utility functions. Thoughts @systay @frouioui @harshit-gangal

Reply:
@GuptaManan100 
The structure for them is quite similar as all of them accept json_val as argument so it was created as one struct.
Info [here](https://dev.mysql.com/doc/refman/8.0/en/json-utility-functions.html)